bazel/api-golden/BUILD.bazel

```diff
@@ -15,9 +15,6 @@ ts_library(
         "index_npm_packages.ts",
         "test_api_report.ts",
     ],
-    # A tsconfig needs to be specified as otherwise `ts_library` will look for the config
-    # in `//:package.json` and this breaks when the BUILD file is copied to `@npm//`.
-    tsconfig = "//dev-infra:tsconfig.json",
```

Comment:
Removing this will break consumption of the dev-infra package within Bazel in general.